crates/node/service/src/service/standard/builder.rs

```diff
@@ -132,7 +132,7 @@ impl RollupNodeBuilder {
         let rpc_client = RpcClient::new(http_hyper, false);
         let l2_provider = RootProvider::<Optimism>::new(rpc_client);
 
-        let rpc_launcher = self.rpc_config.map(|c| c.as_launcher()).unwrap_or_default();
+        let rpc_launcher = self.rpc_config.map(|c| c.as_launcher());
```

Comment:
We'll need to add the `with_healthz` call on the launcher here now since it's optional. We always want the node to expose the `/healthz` endpoint if the rpc is launched so we should call it here.